video/decode/vd_lavc.c

```diff
@@ -143,6 +144,9 @@ static const struct vd_lavc_hwdec *const hwdec_list[] = {
     &mp_vd_lavc_vaapi,
     &mp_vd_lavc_vaapi_copy,
 #endif
+#if HAVE_D3D11VA_HWACCEL
+    &mp_vd_lavc_d3d11va_copy,
+#endif
```

Comment:
What should the ultimate order be?

d3d11va
dxva2
d3d11va-copy
dxva2-copy

? The downside is it interleaves the two requiring duplicated macro guards. But the alternative is that `dxva2-copy` gets preferred over `d3d11va` once that's added.